docs/PaymentRequests.md

```diff
@@ -1,25 +1,52 @@
 # Payment Requests
 
-Payment Requests are a feature which allows BTCPay store owners to create custom invoices which can be shared with others, by sending a link to a counterparty. Funds paid to a payment request use the exchange rate at the time of payment. This allows users to make payments at their convenience without having to negotiate or verify exchange rates with the store owner at the time of payment.
+Payment Requests are a feature which allows BTCPay store owners to create custom invoices which can be shared with others, by sending a link to the custom payment request. Funds paid to a payment request use the exchange rate at the time of payment. This allows users to make payments at their convenience without having to negotiate or verify exchange rates with the store owner at the time of payment.
```

Comment:
> by sending a link to the custom payment request

I never understood this, how is a user sending a link to the custom payment request? Either `of the payment request` or sending a link `to a counterparty`?